### DISCUSSION TITLE
Matt this is a misc thread in case my query was offtopic in the other thread

### DISCUSSION THREAD

**Discussion by SemiAccurate** [anacrolix/torrent#853] [General]
>> Big crypto projects? Odd. Which ones were you referring to? Curious, what do they use anacrolix/torrent for?
> 
> @SemiAccurate I think this is offtopic here. I'm not sure it's appropriate for me to discuss the projects.

@anacrolix if these big crypto projects using your library are sensitive or not public knowledge then sure I don't care to know their names; I didn't think you worked in crypto.

Though what are the crypto projects using anacrolix/torrent for? Like why would they use your library - or even use torrenting libraries in general - if there's nothing among the features of your library that's special for their use cases. I didn't think blockchains used bittorrent protocol for anything.

**Comment by anacrolix** [comment]
I believe they use BitTorrent to bootstrap the blockchain, Bitcoin used to provide torrents for the same purpose, but in this instance it's built in.
